src/main/java/org/sonatype/nexus/ci/iq/ManualApplication.groovy

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2016-present Sonatype, Inc. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package org.sonatype.nexus.ci.iq
+
+import hudson.Extension
+import org.jenkinsci.Symbol
+import org.kohsuke.stapler.DataBoundConstructor
+
+class ManualApplication
+    extends IqApplication
+{
+  @DataBoundConstructor
+  ManualApplication(final String applicationId) {
+    super(applicationId)
+  }
+
+  String getName()
+  {
+    'manual'
+  }
+  @Symbol('manualApplication')
```

Comment:
Right now this symbol isn't used anywhere.

Reply:
> Right now this symbol isn't used anywhere.

Changed 11d7864